demo/__main__.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import logging
+import os
+
+import neo4j
+
+from cartography.config import Config
+from cartography.intel import create_indexes
+from demo.seeds.anthropic import AnthropicSeed
+from demo.seeds.azure import AzureSeed
+from demo.seeds.bigfix import BigfixSeed
+from demo.seeds.cloudlare import CloudflareSeed
+from demo.seeds.crowdstrike import CrowdsrikeSeed
+from demo.seeds.cve import CVESeed
+from demo.seeds.digitalocean import DigitalOceanSeed
+from demo.seeds.duo import DuoSeed
+from demo.seeds.entra import EntraSeed
+from demo.seeds.github import GithubSeed
+from demo.seeds.kandji import KandjiSeed
+from demo.seeds.lastpass import LastpassSeed
+from demo.seeds.openai import OpenAISeed
+from demo.seeds.semgrep import SemgrepSeed
+from demo.seeds.snipeit import SnipeitSeed
+from demo.seeds.tailscale import TailscaleSeed
+
+NEO4J_URL = os.environ.get("NEO4J_URL", "bolt://localhost:7687")
+
+logger = logging.getLogger(__name__)
+
+
+def main(force_flag: bool) -> None:
+    # Set up Neo4j connection
+    neo4j_driver = neo4j.GraphDatabase.driver(NEO4J_URL)
+    neo4j_session = neo4j_driver.session()
+
+    UPDATE_TAG = 0
+
+    # Config
+    config = Config(
+        neo4j_uri=NEO4J_URL,
+    )
+
+    # Check if the database is empty
+    logger.info("Checking if the database is empty...")
+    result = neo4j_session.run(
+        "MATCH (n) WHERE n.lastupdated > 0 RETURN COUNT(n) AS count "
+    )
+    count = result.single()["count"]
+    if count > 0:
+        if force_flag:
+            logger.warning(
+                "Force flag is set. Proceeding to clear the database without confirmation."
+            )
+        else:
+            print(
+                "Database already contains none demo data., Are you sure you want to continue? (y/N)"
+            )
+            answer = input()
+            if answer.lower() != "y":
+                logger.info("Exiting without making any changes.")
+                neo4j_session.close()
+                return
+
+    # Clear the previous database
+    logger.info("Clearing the existing database...")
+    neo4j_session.run("MATCH (n) DETACH DELETE n;")
+
+    # Create indexes
+    logger.info("Creating indexes...")
+    create_indexes.run(neo4j_session, config)
+
+    # Create Human
+    # This is an uggly hack to create a human node for the demo.
+    # We are still working on a better way to build humans (and other abstracts nodes) on Cartography
+    logger.info("Creating human node...")
+    humans = [
+        {"email": "mbsimpson@simpson.corp"},
+        {"email": "hjsimpson@simpson.corp"},
+        {"email": "lmsimpson@simpson.corp"},
+        {"email": "bjsimpson@simpson.corp"},
+    ]
+    neo4j_session.run(
+        "UNWIND $data as item MERGE (h:Human{email: item.email})",
+        data=humans,
+    )
```

Comment:
@jychp thoughts on bringing abstract node types into this demo at the same time that we bring them into Cartography itself? I feel like bringing the Human node type into this demo now doesn't do it justice and may confuse users at this time

Reply:
+1, let's hold off on adding Human nodes until then